scraper/github/__init__.py

```diff
@@ -73,7 +77,7 @@ def _num_requests_needed(num_repos, factor=2, wiggle_room=100):
     return num_repos * factor + wiggle_room
 
 
-def _check_api_limits(gh_session, api_required=250, sleep_time=15):
```

Comment:
Rather than removing it here, should the `time.sleep(10)` line at the end of this function be updated to use this `sleep_time` argument?

Reply:
So I was on the fence about which way to go on that decision. When I looked into the blame I found that even when this function was [first added](https://github.com/LLNL/scraper/commit/0a641df1a6504205fdfb395fc3b509c1130456d9) the argument was never used. Looking at the code I feel like using the value of `time_to_reset` makes the most sense as we already calculate a specific time until the API limit is reset. I was trying to make no changes to functionality with this PR so I just went with removing the unused argument in this case.